### ISSUE TITLE
Please provide a coreos helm server

### ISSUE THREAD

**Issue by gianrubio** [coreos/prometheus-operator#660]
The current helm repo is broken because they're not update after PR. I know that @mgoodness 
 is busy with other stuffs but we can't keep the charts outdated. Also I trust more in a coreos helm repo than in repo managed by users. 

Any plans to coreos provide a helm server, so we could sync all the changes with the helm chart upstream? 

```
$ helm install opsgoodness/prometheus
Error: apiVersion "monitoring.coreos.com/v1alpha1" in prometheus/templates/prometheus.yaml is not available
```
#592

**Comment by brancz** [comment]
I'm happy to have a repo hosted by us with the quay helm registry. If I remember correctly, it requires an additional fetch step for users, which we can document.

Currently I have no resources to invest into helm things, so I don't see it being automated via CI any time soon. I'm happy to do what it takes to enable this though. If I create the repositories, would you be interested in keeping them up to date?

**Comment by gianrubio** [comment]
>  If I create the repositories, would you be interested in keeping them up to date?

Yes, I'm willing to maintain the helm repo and work to automatize the CI stuff. I'm going to start figuring out how the helm server works so I'll back with further instructions.

**Comment by brancz** [comment]
Let me know what I can do and we'll make it happen :slightly_smiling_face: .

**Comment by atopuzov** [comment]
I can chip in as well as I use the charts to deploy the operator and kube-prom trough helm.

**Comment by gianrubio** [comment]
Fixed on #776
